Anlab.Core/Models/EmailSettings.cs

```diff
@@ -10,5 +10,6 @@ public class EmailSettings
         public string Password { get; set; }
         public string Host { get; set; }
         public int Port { get; set; }
+        public string BccAnlab { get; set; }

```

Comment:
@jSylvestre please change to Anlab or AnlabAddress or something. doesn't need to be bcc specific